applications/MPMApplication/python_scripts/kratos_main_mpm.py

```diff
@@ -1,10 +1,10 @@
 
```

Comment:
Note that if you follow the naming convention you can use the standard main file as the other applications do.

Reply:
[MainKratos.zip](https://github.com/user-attachments/files/16314422/MainKratos.zip)
 I mean this one.

Reply:
This is exactly the reason why I'm changing the name from `MPMAnalysis` to `MpmAnalysis`. So I think I can also remove the script `kratos_main_mpm.py` from the application, right?

Reply:
sorry @rubenzorrilla, can you confirm me that everything is ok before merging?

Reply:
Everything looks good to me. Thanks for taking my comments into consideration.